.travis.yml

```diff
@@ -8,6 +8,10 @@ rvm:
 - 2.0.0
 - 2.1.0
 - 2.2.0
+- 2.2.2
+- 2.2.3
```

Comment:
Testing against 2.2.3 is not needed, otherwise the test matrix will become to large.